packages/core/template/portfolio/contents/project2.md

```diff
@@ -0,0 +1,23 @@
+<frontmatter>
+  title: Project 2
+</frontmatter>
+
+<br>
+
+### Project 2
+
+<div class="card mb-3">
+  <h3 class="card-header">Name of project</h3>
+  <div class="card-body">
+    <h5 class="card-title">Short description of project</h5>
+  </div>
+  <pic src="../contents/assests/MarkbindExample.png" width="500" alt="project-display"/>
```

Comment:
What if we switched this from a pic to a youtube embed? Just so the projects look distinctly different. You can also replace the example text underneath to include info about embeds, and the example text under the first text to include links to pic + annotations